<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KWStartupWidget</class>
 <widget class="QWidget" name="KWStartupWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>382</width>
    <height>234</height>
   </rect>
  </property>
  <layout class="QVBoxLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QTabWidget" name="tabs">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="documentMode">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="sizeTab">
        <attribute name="title">
         <string>Page Size &amp;&amp; Margins</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="columnsTab">
        <attribute name="title">
         <string>Columns</string>
        </attribute>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QFrame">
       <property name="frameStyle">
        <enum>QFrame::VLine</enum>
       </property>
       <property name="fixedWidth">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="previewPane" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame">
     <property name="frameStyle">
      <enum>QFrame::HLine</enum>
     </property>
     <property name="fixedHeight">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="createButtonLayout">
     <property name="leftMargin">
      <number>6</number>
     </property>
     <property name="topMargin">
      <number>6</number>
     </property>
     <property name="rightMargin">
      <number>6</number>
     </property>
     <property name="bottomMargin">
      <number>6</number>
     </property>
     <item>
      <widget class="QPushButton" name="createButton">
       <property name="text">
        <string>Create</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabs</tabstop>
  <tabstop>createButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
